src/Question.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { Link, BrowserRouter as Router } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+class Question extends Component {
```

Comment:
Your all comonents are implemented by using ES6 class. Why not to use the stateless functional component where is possible?